etc/config/catchuparchiver_config.yaml

```diff
@@ -0,0 +1,41 @@
+# Configuration settings for CatchupArchiver
+ROOT:
+  REDIS_HOST: localhost
+  ARCHIVER_REDIS_DB: 14
+  FORWARDER_PUBLISH_QUEUE: catchup_foreman_ack_publish
+  ARCHIVE_CTRL_PUBLISH_QUEUE: archive_ctrl_publish
+  ARCHIVE_CTRL_CONSUME_QUEUE: archive_ctrl_consume
```

Comment:
Is this going to interfere with ComCam communications?

Reply:
This won't interfere with ComCam because this is running on different vhost.